packages/bbui/src/helpers.js

```diff
@@ -166,7 +166,7 @@ export const stringifyDate = (
   const offsetForTimezone = (enableTime && ignoreTimezones) || timeOnly
   if (offsetForTimezone) {
     // Ensure we use the correct offset for the date
-    const referenceDate = timeOnly ? new Date() : value.toDate()
+    const referenceDate = value.toDate()
```

Comment:
I am at gmt+2 and without this, every time I called this function, one extra hour was added to the result. I will try to write unit tests to ensure its behaviour

Reply:
Was this when using a real time type in a postgres DB? This would probably happen if the backing datasource behaved differently than we currently expect compared to using a non-timezone aware timestamp type. I'm not sure a unit test will help here because I'm almost certain it's to do with what the value looks like when it gets saved into the external DB and then returned to the frontend.

Reply:
This is probably just more evidence that a "time only" data type really needs to be its own thing, rather than trying to store it inside an ISO date string.

Reply:
I agree that a time only is required. Also in the backend, having the same type with different data shape in the db is also pretty hard to manage

Reply:
The issue happened fetching time columns from dbs or "creating" internal db timeOnly fields. Running a deeper testing